lib/tasks/newrelicyml.rake

```diff
@@ -0,0 +1,13 @@
+# This file is distributed under New Relic's license terms.
+# See https://github.com/newrelic/newrelic-ruby-agent/blob/main/LICENSE for complete details.
+# frozen_string_literal: true
+
+require_relative './helpers/newrelicyml'
+
+namespace :newrelic do
+  desc 'Update newrelic.yml with latest config options from default_source.rb'
+  task :update_newrelicyml do
+    NewRelicYML.write_file
+    puts 'newrelic.yml updated'
```

Comment:
Hmm I guess if there was something put into `default_source` that wasn't expected. Are you thinking I should add a `rescue`?

Reply:
I think we would want to know if it fails though, that way we can address it. If we rescue it, then it would just silently fail, which doesn't seem ideal.

Reply:
Agree Tanna! Is this fine as is then?

Reply:
I think so. I think unless there is a specific error we would want to have logic around handling, the error failing the task seems good to me.

Reply:
Yep, you're both right. I'm being paranoid. This is fine.